libraries/ESP32/examples/Zigbee/Zigbee_Light_Bulb/README.md

```diff
@@ -39,6 +39,9 @@ You can do the following:
 * In the Arduino IDE go to the Tools menu and set `Erase All Flash Before Sketch Upload` to `Enabled`
 * In the sketch uncomment function `esp_zb_nvram_erase_at_start(true);` located in `esp_zb_task` function.
 
+By default, the coordinator network is open for 180s after rebooting or flashing new firmware. After that, the network is closed for adding new devices.
```

Comment:
flashing new the firmware...

Reply:
new the what? 😄 The sentence looks good to me as-is